docs/src/pages/docs/en/design-principles.md

```diff
@@ -4,26 +4,26 @@ description: Design Principles
 layout: ../../../layouts/MainLayout.astro
 ---
 
-## Components should feel like native HTML
+## Elements should feel like native HTML
 
 Web components allow us to "extend the browser", and in this project we take that seriously, to the degree of aligning the design of the components to the [design of native HTML elements](https://www.w3.org/TR/design-principles/). In some cases this can seem less ideal/progressive, but it creates an intutive and predictable interface for anyone familiar with HTML concepts.
 
-## Element Naming
+## Elements should be named consistently
 
 - Prefix with `media-` for scoping.
 - The suffix should make it clear what the primary user interaction is with the element, mapping to the matching native HTML element when possible. e.g. `-button`, `-range`, `-image`.
   - Use `-container` for elements that are purely for laying out other elements. More specific layout names like `-bar` are acceptible when the name is clear and intentional.
   - Use `-display` when the primary use is displaying a state or data detail. e.g. `media-duration-display`.
 
-## Element Independence
+## Elements should be independently usable
 
-A goal of this project is that each UI element could be used independently of the project, in the same way an HTML Button can be used independently of an HTML Form. This will make it possible for other players to share these base elements.
+A goal of this project is that each UI element could be used independently of the project, in the same way an HTML Button can be used independently of an HTML Form. This will make it possible for other players to share these base elements, and open the door for unpredicable uses.
```

Comment:
```suggestion
A goal of this project is that each UI element could be used independently of the project, in the same way an HTML Button can be used independently of an HTML Form. This will make it possible for other players to share these base elements, and open the door for unpredictable uses.
```